### ISSUE TITLE
4.4.7 crashes in postinstall

### ISSUE THREAD

**Issue by lerouxb** [sinonjs/sinon#1743]
```
Error: Cannot find module '/blaaaah/node_modules/sinon/scripts/support-sinon.js'
    at Function.Module._resolveFilename (module.js:543:15)
    at Function.Module._load (module.js:470:25)
    at Function.Module.runMain (module.js:690:10)
    at startup (bootstrap_node.js:194:16)
    at bootstrap_node.js:666:3
```

**Comment by bvanderlaan** [comment]
I'm seeing that too

**Comment by andrewaustin** [comment]
This started failing for me as well.

**Comment by sarunas** [comment]
Same here

**Comment by JanFoerste** [comment]
Same problem here

**Comment by simranjeet2589** [comment]
same here

**Comment by mroderick** [comment]
I am looking into this

**Comment by AmirMaurice** [comment]
Same here for me

**Comment by leoner** [comment]
me too.

**Comment by andnp** [comment]
Yup... I think this is consistent for all users, and I believe @mroderick is looking into it, so we probably don't need any more "me too"'s here. If anything, you can always thumbs up the initial post to reduce comment noise.

**Comment by lerouxb** [comment]
Thanks for the quick response!

**Comment by mroderick** [comment]
I have fixed this in a new release `sinon@4.4.8`, sorry for the disturbance folks!
